application/Program.cs

```diff
@@ -10,17 +14,30 @@ public class Program
 	{
 		public static void Main(string[] args)
 		{
-            using (var amz = new AmazonSynchronizer(new Configuration()))
+            var config = new Configuration();
+            var telemetry = new TelemetryClient();
+		    telemetry.InstrumentationKey = config.InstrumentationKey;
+		    telemetry.Context.Device.Id = Environment.MachineName;
+            telemetry.Context.Session.Id = Guid.NewGuid().ToString();
+		    telemetry.Context.Location.Ip = Dns.GetHostEntryAsync(Dns.GetHostName())
```

Comment:
I don't think so, it's not like the asp version.

https://azure.microsoft.com/en-us/documentation/articles/app-insights-windows-desktop/